app/views/helpers/PointerRecord.scala

```diff
@@ -16,18 +16,33 @@
 
 package views.helpers
 
+import connectors.CodeListConnector
 import controllers.declaration.routes._
 import forms.declaration.ModeOfTransportCode
 import models.ExportsDeclaration
 import models.declaration.{Container, ExportItem}
+import play.api.i18n.Messages
 import play.api.mvc.Call
+import services.{Countries, DocumentTypeService, PackageTypesService}
 
 trait PointerRecord {
-  def fetchValue(dec: ExportsDeclaration, args: Int*): Option[String]
+  def fetchRawValue(dec: ExportsDeclaration, args: Int*): Option[String]
+  def fetchReadableValue(
+    dec: ExportsDeclaration,
+    args: Int*
+  )(implicit msgs: Messages, countryHelper: CountryHelper, codeListConnector: CodeListConnector): Option[String]
   val pageLink1Param: Option[Call] = None
   val pageLink2Param: Option[String => Call] = None
 }
 
+abstract class DefaultPointerRecord extends PointerRecord {
```

Comment:
This could still be a `Trait`, isn't it?

Reply:
I only added this abstract class to get around the annoying issue of "parameter x method is never used" error if I left the default implementation in the trait. For some reason Scala does not like you defining default method implementations in traits where you don't use all the method parameters, but method implementations in abstract classes it has no problems with! :shrug: